### ISSUE TITLE
Support for generator methods?

### ISSUE THREAD

**Issue by anba** [ljharb/is-generator-function#2]
`isGeneratorFunction({*g(){}}.g)` reports `false` in V8.

Current toString output for generator methods in V8 (at v8/v8-git-mirror@7611aace3e7bf6a3a25deb899ff2dc3bab1a8e66):
```
d8> ({*g(){}}.g).toString()
"*g(){}"
d8> "use strict"; (class {*g(){}}.prototype.g).toString()
"*g(){}"
d8> "use strict"; (class {static *g(){}}.g).toString()
"*g(){}"
```

**Comment by ljharb** [comment]
ah, I didn't account for concise generator methods - thanks :-) Which engines (Chrome, node, io.js) support a v8 with concise generator methods?

**Comment by anba** [comment]
They're only enabled when manually adding the `--es_staging` flag.

**Comment by ljharb** [comment]
Released as `v1.0.4`. This is a patch change since concise methods are not in any actual released engines yet.
